pkg/webhooks/jax/jaxjob_webhook.go

```diff
@@ -71,8 +70,9 @@ func (w *Webhook) ValidateDelete(context.Context, runtime.Object) (admission.War
 
 func validateJAXJob(job *trainingoperator.JAXJob) field.ErrorList {
 	var allErrs field.ErrorList
-	if errors := apimachineryvalidation.NameIsDNS1035Label(job.ObjectMeta.Name, false); len(errors) != 0 {
-		allErrs = append(allErrs, field.Invalid(field.NewPath("metadata").Child("name"), job.Name, fmt.Sprintf("should match: %v", strings.Join(errors, ","))))
+	err := utils.ValidateJobName(job.ObjectMeta.Name)
```

Comment:
This commonize does not have value since we already use the Kube validations lib here.
My expectations are commonizing replicaSpec validations: https://github.com/kubeflow/training-operator/blob/7c5ea70f10d07732826907e75d1e5a50db97a059/pkg/webhooks/pytorch/pytorchjob_webhook.go#L93-L109